tracing.go

```diff
@@ -109,3 +110,14 @@ func (s Span) NewChildSpan() *Span {
 	}
 	return childSpan
 }
+
+func (s *Span) sampleRootSpan(sampleRate float64) {
+	// Sampling only affects root spans
+	if s.ParentID() != 0 {
+		return
+	}
+
+	if rand.Float64() > sampleRate {
```

Comment:
I use > since it's to decide whether to *not* sample.

If the sampleRate is 0.3, then we don't want to sample most of the time, which is when the random float is > 0.3